libs/eventually-broker/src/stores/seed.ts

```diff
@@ -21,7 +21,7 @@ create table if not exists public.subscriptions (
   consumer varchar(100) not null,
   path varchar(100) not null,
   streams varchar(100) not null,
-  names varchar(500) not null,
+  names varchar(750) not null,
```

Comment:
Not blocker: I guess this column holds all the consumer names, right? I don't think it would be bad to make this unlimited.

Reply:
Correct, 750 chars seems a little bit excesive, but fine.